flow-typed/Claim.js

```diff
@@ -191,32 +191,3 @@ declare type ChannelPublishParams = {
   tags?: Array<string>,
   languages?: Array<string>,
 };
-
-declare type CollectionUpdateParams = {
```

Comment:
:small_red_triangle: thumbs not appearing despite resolved (or should have been resolved)

I have a bunch of playlist, all only containing 1 same video.  After F5, the thumbs are not appearing for all of them except 1.  It's the same claim, so if it appeared for 1 playlist thumb, it should have appeared for all.


https://user-images.githubusercontent.com/64950861/189807268-e9d9dff7-8d92-44c5-acca-dd57a2b3b0d0.mp4